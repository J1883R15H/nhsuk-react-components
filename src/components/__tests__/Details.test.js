import React from 'react';
import { shallow } from 'enzyme';
import Details from '../details';

describe('Details', () => {
  it('normal matches snapshot', () => {
    const shallowMount = shallow(<Details text="Test Text">Test Text</Details>);
    expect(shallowMount).toMatchSnapshot();
    shallowMount.unmount();
  });
  it('expander matches snapshot', () => {
    const shallowMount = shallow(
      <Details expander text="Test Text">
        Test Text
      </Details>
    );
    expect(shallowMount).toMatchSnapshot();
    shallowMount.unmount();
  });
});
