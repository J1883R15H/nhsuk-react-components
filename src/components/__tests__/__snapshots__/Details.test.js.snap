// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Details expander matches snapshot 1`] = `
<details
  className="nhsuk-details nhsuk-expander"
  style={Object {}}
>
  <summary
    className="nhsuk-details__summary"
  >
    <span
      className="nhsuk-details__summary-text"
    >
      Test Text
    </span>
  </summary>
  <div
    className="nhsuk-details__text"
  >
    Test Text
  </div>
</details>
`;

exports[`Details normal matches snapshot 1`] = `
<details
  className="nhsuk-details"
  style={Object {}}
>
  <summary
    className="nhsuk-details__summary"
  >
    <span
      className="nhsuk-details__summary-text"
    >
      Test Text
    </span>
  </summary>
  <div
    className="nhsuk-details__text"
  >
    Test Text
  </div>
</details>
`;
